### PR TITLE
ui: slow down retries after network errors

### PR DESCRIPTION
**Pull request by maxlang**
Temporary fix for #8699

cc @cockroachdb/stability

---

This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/9630)

### CODE REVIEW

File: ui/app/redux/cachedDataReducer.ts [hunk 1]
```diff
@@ -160,7 +160,10 @@ export class CachedDataReducer<TRequest, TResponseMessage> {
         dispatch(this.receiveData(data, req));
       }).catch((error: Error) => {
         // If an error occurred during the fetch, it to the store.
```
Comment (tschottdorf) [inline ui/app/redux/cachedDataReducer.ts]: verb missing?

**Review by tbg — APPROVED** [review]
Assuming s/o from ui team reviews.

File: ui/app/redux/cachedDataReducer.ts (hunk 1 above)
Comment (maxlang) [inline ui/app/redux/cachedDataReducer.ts]: Done.

**Comment by mrtracy** [comment]
Seems like a fair temporary fix; clever even.  LGTM.

---

Review status: 0 of 2 files reviewed at latest revision, 1 unresolved discussion, all commit checks successful.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/cockroach/9630#-:-KSwngMiZ1pKPe7t7jSt:b-nghc30)*

**Pull request merged**